### ISSUE TITLE
Panic when parsing a particular HTML string

### ISSUE THREAD

**Issue by ephraimkunz** [jugglerchris/rust-html2text#61]
Hey, I noticed the following code panics:
```rust
    let html = r#"<table><td><p data-aid="133347338" id="p5">3,266</p>"#;
    let decorator = html2text::render::text_renderer::PlainDecorator::new();
    let text = html2text::from_read_with_decorator(html.as_bytes(), usize::MAX, decorator.clone());
    println!("{}", text);
```

`thread 'main' panicked at 'attempt to multiply with overflow', /Users/ephraimkunz/.cargo/registry/src/github.com-1ecc6299db9ec823/html2text-0.4.2/src/lib.rs:1408:29`

**Comment by jugglerchris** [comment]
Hi,
Thanks for the report!
This would have happened with any HTML containing a table when passing a width close to `usize::MAX`.
While using a huge width wasn't how I expected this to be used (the intention was to format for a fixed-width terminal), this isn't the first time and I think this should be supported.  :-)  (See #53 as well).

**Comment by jugglerchris** [comment]
This is now fixed, and released in 0.4.3.
